--- extra/makefile~	2006-09-25 06:17:22.000000000 +0200
+++ extra/makefile	2013-06-01 22:44:59.000000000 +0200
@@ -57,7 +57,7 @@
 
 .SUFFIXES: .pd_linux
 
-LINUXCFLAGS = -DPD -O2 -funroll-loops -fomit-frame-pointer -fPIC -m32 \
+LINUXCFLAGS = -DPD -O2 -funroll-loops -fomit-frame-pointer -fPIC \
     -Wall -W -Wshadow -Wstrict-prototypes \
     -Wno-unused -Wno-parentheses -Wno-switch $(CFLAGS)
 
@@ -65,7 +65,7 @@
 
 .c.pd_linux:
 	$(CC) $(LINUXCFLAGS) $(LINUXINCLUDE) -o $*.o -c $*.c
-	$(CC) -m32 -export_dynamic -shared -o $*.pd_linux $*.o -lc -lm
+	$(CC) -export_dynamic -shared -o $*.pd_linux $*.o -lc -lm
 	strip --strip-unneeded $*.pd_linux
 	rm -f $*.o
 
--- extra/expr~/makefile~	2006-09-25 06:19:11.000000000 +0200
+++ extra/expr~/makefile	2013-06-01 22:45:27.000000000 +0200
@@ -77,7 +77,7 @@
 LINUXOBJ =  vexp.pd_linux_o vexp_fun.pd_linux_o vexp_if.pd_linux_o
 .SUFFIXES: .pd_linux_o
 
-LINUXCFLAGS = -DPD -O2 -funroll-loops -fomit-frame-pointer -fPIC -m32 \
+LINUXCFLAGS = -DPD -O2 -funroll-loops -fomit-frame-pointer -fPIC \
     -Wall -W -Wshadow -Wstrict-prototypes \
     -Wno-unused -Wno-parentheses -Wno-switch 
 
@@ -87,7 +87,7 @@
 	$(CC) -g $(LINUXCFLAGS) $(LINUXINCLUDE) -o $*.pd_linux_o -c $*.c
 
 expr.pd_linux: $(LINUXOBJ)
-	$(CC) -m32 -export_dynamic  -shared -o expr.pd_linux $(LINUXOBJ) -lc -lm
+	$(CC) -export_dynamic  -shared -o expr.pd_linux $(LINUXOBJ) -lc -lm
 	strip --strip-unneeded expr.pd_linux
 
 expr~.pd_linux: expr.pd_linux
